src/lib/libpthread.js

```diff
@@ -42,6 +42,15 @@ const pthreadWorkerScript = TARGET_JS_NAME;
 // See https://github.com/emscripten-core/emscripten/issues/22394
 const pthreadWorkerOptions = `{
 #if EXPORT_ES6
+#if MIN_FIREFOX_VERSION < 114
+#error new Worker() supports ECMAScript module only starting from Firefox 114. Pass -sMIN_FIREFOX_VERSION=114 to target -sEXPORT_ES6 with -pthread. See https://caniuse.com/mdn-api_worker_worker_ecmascript_modules
+#endif
+#if MIN_CHROME_VERSION < 80
+#error new Worker() supports ECMAScript module only starting from Chrome 80. Pass -sMIN_CHROME_VERSION=80 to target -sEXPORT_ES6 with -pthread. See https://caniuse.com/mdn-api_worker_worker_ecmascript_modules
+#endif
+#if MIN_SAFARI_VERSION < 150000
+#error new Worker() supports ECMAScript module only starting from Safari 15. Pass -sMIN_SAFARI_VERSION=150000 to target -sEXPORT_ES6 with -pthread. See https://caniuse.com/mdn-api_worker_worker_ecmascript_modules
+#endif
```

Comment:
It is even possible to hit these error now that the feature is added?   i.e. `-sEXPORT_ES6` + `-pthreads` will now force the correct versions or fail, right?